<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="HostListWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>41</y>
      <width>281</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="b_AddHost">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>610</y>
      <width>120</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Host</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_DeleteHost">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Host</string>
    </property>
   </widget>
   <widget class="QFrame" name="framePlot">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>701</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="l_HostName">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>510</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Host Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_Address">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>510</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Address:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_HostName">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>510</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_Address">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>510</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_PingHint">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>550</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ping (ms):</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_Average">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>580</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Average:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_Average">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>580</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_StdDev">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>580</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Standard deviation:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_StdDev">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>580</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_Min">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>580</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_Min">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>580</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_Max">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>580</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_Max">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>580</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_Lost">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>620</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Packets lost (percent):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_lost">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>620</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_TTL">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>620</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time To Live:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_TTL">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>620</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="Menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="Status"/>
 </widget>
 <resources/>
 <connections/>
</ui>
